bin/package

```diff
@@ -76,12 +78,12 @@ cd bin
 echo "Building for ${TARGET_ENVIRONMENT}..."
 
 # Make sure the correct version of pyinstaller and setuptools are installed
-pip install -U "PyInstaller==3.2.1"
+pip install -U "PyInstaller"
```

Comment:
Can we keep the pin here with the newer version? Assuming it's a newer version.

Reply:
I'm still not sure about removing this pin, but I'm okay with merging either way.